app/src/main/java/com/github/mateo762/myapplication/habits/fragments/ListFragment.kt

```diff
@@ -18,11 +20,17 @@ private const val ARG_PARAM2 = "param2"
  * create an instance of this fragment.
  */
 class ListFragment : Fragment() {
+
     // TODO: Rename and change types of parameters
     private var param1: String? = null
     private var param2: String? = null
 
```

Comment:
Nice that you added this advertisement, maybe mention that you added this with the group